cli/cmd/sources.go

```diff
@@ -230,7 +231,7 @@ func parseSourceLabelFlags() (string, string, string, labels.Set) {
 
 func init() {
 	sourceFlags = pflag.NewFlagSet("sourceFlags", pflag.ContinueOnError)
-	sourceFlags.StringVarP(&namespaceFlag, namespaceFlagName, "n", "default", "Kubernetes Namespace for Source")
```

Comment:
reviewer note: the `namespaceFlag` is defined in `install` command and this line overrides it's default value `odigos-system` with `default` which leaks into the `install`.